src/app/child-dev-project/attendance/child-attendance/child-attendance.component.spec.ts

```diff
@@ -156,33 +155,30 @@ const ATTENDANCE_ENTITIES: AttendanceMonth[] = (() => {
   return data;
 })();
 
-
```

Comment:
I do like the opportunity to add 2-3 empty lines separating different logical blocks